tests/farmers.test.js

```diff
@@ -69,7 +69,31 @@ describe('Farmer route', () => {
           done(err);
         });
     });
-    it('It deletes farmer details', done => {
+    it('It should return an array of farmers', (done) => {
+      chai
+        .request(server)
+        .get('/api/v1/farmers')
+        .set('Authorization', token)
+        .end((err, res) => {
+          res.should.have.status(200);
+          res.body.should.be.a('object');
+          res.body.message.should.equal('Farmers records found');
+          done(err);
+        });
+    });
+    it('It should return an array of farmers', (done) => {
```

Comment:
There's a typo.
`It should return an array of farmers` should be `It should return a single farmer`

Reply:
Done